templates/vm-import-operator/VERSION/operator.yaml.in

```diff
@@ -185,6 +185,8 @@ spec:
               value: "vm-import-operator"
             - name: OS_CONFIGMAP_NAME
             - name: OS_CONFIGMAP_NAMESPACE
+            - name: IMPORT_POD_RESTART_TOLERANCE
```

Comment:
It should also be added to pkg/operator/resources/operator/operator.go createControllerEnv

Reply:
One more comment - the env variable in this resource will be ignored, since it will be set to the vm-import-operator deployment resource. therefore it can be removed.